assignment4/day 5/main.js

```diff
@@ -0,0 +1,45 @@
+var secondLargestModule = require("./files/secondLargest");
+var calculateFrequencyModule = require("./files/calculateFrequency");
+var flattenObjectModule = require("./files/flatten");
+var unflattenObjectModule = require("./files/unflatten");
+var output;
+
+function secondLargest() {
+  var inputs = [[3,56,7,32,9,14],[21, 87, 91, 31, 97, 59, 8, 83, 27, 23, 76, 72, 31, 91, 65, 77, 84, 27, 56, 59],[21, 87, 91, 31, 97, 59, 8, 83, 27, 23, 76, 72, 31, 91, 65, 77, 84, 92, 56, 59]];
+  for (var i = 0; i < inputs.length; i++) {
+	output = secondLargestModule.secondLargest(inputs[i]);
+	showInputOutput(inputs[i],output);
+  }
+}
+
+function frequencyCalculation() {
+  var inputs = ["dummy","dummy text","shali is running fast","shali is running fast","u@#d$fkj$#!kds"];
+  for (var i = 0; i < inputs.length; i++) {
```

Comment:
instead of doing this loop again n again wrap this logic inside a function that takes _input_ and _function to execute_ as arguments.